aws/createResources.sh

```diff
@@ -152,19 +153,19 @@ createS3Bucket() {
 
 uploadS3Bucket() {
     # Add the ‘website’ configuration and bucket policy
-    aws s3 website s3://$BUCKET_NAME/ --index-document index.html --error-document index.html  --region $REGION
+    $aws_cmd s3 website s3://$BUCKET_NAME/ --index-document index.html --error-document index.html  --region $REGION
     cat s3-bucket-policy.json | sed 's/BUCKET_NAME/'$BUCKET_NAME'/' > /tmp/s3-bucket-policy.json
-    aws s3api put-bucket-policy --bucket $BUCKET_NAME --policy file:///tmp/s3-bucket-policy.json  --region $REGION
+    $aws_cmd s3api put-bucket-policy --bucket $BUCKET_NAME --policy file:///tmp/s3-bucket-policy.json  --region $REGION
     #Build the project and sync it up to the bucket
     if [ ! -d "$NPM_DIR" ]; then
         npm install
     fi
     cd ..
     echo "Building the project"
-    ng build
+    ng build $( if [ "$aws_cmd" == "awslocal" ]; then echo "--base-href /$BUCKET_NAME/"; fi )
```

Comment:
This is required because local S3 storage uses the bucket name as path prefix (because bucket domain names cannot be used locally without a DNS server or entries in `/etc/hosts`).